packages/wonder-blocks-color/index.js

```diff
@@ -31,9 +31,14 @@ const Color = {
     pink: "#fa50ae",
 };
 
+const SemanticColor = {
+    controlDefault: Color.blue,
+    controlDestructive: Color.red,
+};
+
 // Note: Only blue and red are valid tints for Buttons IconButtons. We
```

Comment:
🛑 This should probably be removed, right?

🤷‍♂️ It might be worth replacing this with something like `export type ValidSemanticColors = "default" | "destructive";` and then you can use that in your button code.